Mathlib/Analysis/SpecialFunctions/Pow/Real.lean

```diff
@@ -513,6 +513,31 @@ lemma rpow_mul_intCast (hx : 0 ≤ x) (y : ℝ) (n : ℤ) : x ^ (y * n) = (x ^ y
 /-! Note: lemmas about `(∏ i in s, f i ^ r)` such as `Real.finset_prod_rpow` are proved
 in `Mathlib/Analysis/SpecialFunctions/Pow/NNReal.lean` instead. -/
 
+/-- `rpow` version of `Finset.prod_pow_eq_pow_sum`. -/
+theorem finset_prod_rpow_eq_rpow_sum
+    {ι : Type*} (s : Finset ι) {f : ι → ℝ} (hs : ∀ i ∈ s, 0 ≤ f i) {r : ℝ} (hr : 0 ≤ r) :
+    ∏ i in s, r ^ f i = r ^ (∑ i in s, f i) := by
```

Comment:
This already exists as [Real.rpow_sum_of_nonneg](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Analysis/SpecialFunctions/Pow/Real.html#Real.rpow_sum_of_nonneg)